### ISSUE TITLE
dragon

### ISSUE THREAD

**Issue by AZ1229-forever** [romanlee6/langground#3]
Hello, I'm very interested in your work. While running the code in the Dragon environment, the performance seems inconsistent with what was demonstrated in your paper. I suspect there might be issues with my parameter settings. Could you please review the specific parameters below to check if any settings are incorrect? Thank you very much.

Namespace(num_epochs=5000, epoch_size=10, batch_size=128, nprocesses=0, hid_size=256, recurrent=True, gamma=0.95, tau=1.0, seed=13, normalize_rewards=False, lrate=0.001, entr=0, value_coeff=0.01, env_name='mini_dragon', max_steps=100, nactions='1', action_scale=1.0, plot=True, plot_env='main', save='trained_models', save_every=100, load='trained_models', display=False, random=False, commnet=1, ic3net=True, timmac=False, mac=False, nagents=3, comm_mode='avg', comm_passes=1, comm_mask_zero=False, mean_ratio=0, rnn_type='LSTM', detach_gap=10, comm_init='uniform', hard_attn=1, comm_action_one=False, comm_action_zero=False, advantages_per_action=False, share_weights=False, log_dir='tb_logs', exp_name='mini_dragon_supervised_exact_norm_10_large_4.25', use_proto=False, discrete_comm=False, num_proto=6, add_comm_noise=True, comm_dim=256, gating_head_cost_factor=0.0, restore=False, gate_reward_curriculum=True, variable_gate=False, optim_name='RMSprop', scheduleLR=False, budget=1.0, use_tj_curric=False, soft_budget=1.0, load_pretrain=False, pretrain_exp_name=None, min_comm_loss=False, eta_comm_loss=1.0, max_info=False, eta_info=1.0, autoencoder=False, autoencoder_action=False, remove_null=False, null_dict_dir='', map_name='3m', mha_comm=False, num_heads=1, preencode=False, vae=False, use_vqvib=False, use_compositional=False, contrastive=False, comp_beta=0.01, supervised_comm=True, norm_comm=True, data_path='../../LLM/embedded_256_offline_llm_dataset_dragon_1500_4,25.csv', supervised_gamma=10.0, sampling_method='exact', prey_loc_res=False, eval_mode='eval', nfriendly=3, num_actions=[9, 2], dim_actions=2, num_inputs=36, continuous=False, naction_heads=[9, 2])
using commnet
